docs/source/kernel-kubernetes.md

```diff
@@ -269,6 +275,9 @@ spec:
             value: "300"
           - name: KIP_PULL_POLICY
             value: "IfNotPresent"
+          # Authorization token required for all requests (should match EG_AUTH_TOKEN)
```

Comment:
Same comment applies here, although the reminder that this value should match `EG_AUTH_TOKEN` should be kept.